controllers/suggestions/suggest_apiwrite_controller.py

```diff
@@ -1,10 +1,11 @@
 from consts.auth_type import AuthType
 from controllers.base_controller import LoggedInHandler
+from controllers.suggestions.suggestions_base_controller import SuggestionsBaseController
 from helpers.suggestions.suggestion_creator import SuggestionCreator
 from template_engine import jinja2_engine
 
 
-class SuggestApiWriteController(LoggedInHandler):
+class SuggestApiWriteController(SuggestionsBaseController):
```

Comment:
If I understand correctly, SuggestionsBaseController is only useful to wrap the email sender. Would it make more sense to keep this LoggedInHandler and wrap the email sending in a helper?